orttraining/orttraining/python/training/onnxblock/sample/model_generation.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# model_generation.py
+
+import argparse
+import os
+
+import onnx
+
+import onnxruntime.training.onnxblock as onnxblock
+
+
+class MyTrainingModel(onnxblock.TrainingModel):
+    def __init__(self):
+        super().__init__()
+        self._loss = {
+            "MSELoss": onnxblock.loss.MSELoss(),
```

Comment:
if this is a sample code, I think reader would be confused why offline requires 3 loss functions? In most cases, I think single loss is needed, right?